.circleci/README.md

```diff
@@ -0,0 +1,81 @@
+PyNE-CI Documentation
+=====================
+
+
+PyNE Circle-CI use CIrcle_CI version 2.1 and is composed in 4 main sections:
+`executors`, `commands`, `jobs` and `workflow`.
+
+Executors
+---------
+The `executors` define aliases to the environment to run the different jobs, in our case
+different docker images containing different combinations of the optional
+dependencies of PyNE (`MOAB`, `pyMOAB`, `DAGMC`) in one of the two supported
+PyTHON flavors (Python 2.7 or Python 3.6).
+
+
+Commands
+---------
+The `commands` section allows the definition of macro commands with arguments to be used
+in the different jobs.
+
+`news_check`: check the presence of a new news file in the news folder
+
+
+`save_container`: saves the status of a container to be reloaded in an
+other job.
+
+    - `arguments`: 
+        - `build` (string): build configuration parameters, used to identify a
+          build configuration
+
+
+`pull_container`: pulls a previously saved (using the `save_container` command) status of container.
+
+    - `arguments`:
+        - `build` (string): build configuration parameters (has to match the build
+          argument used to save the container)
+
+
+`checkout_build`: checks out PyNE branch to be tested, builds it and saves it
+using the `save_container` command.
+    
+    - `arguments`:
+        - `build` (string): used to identify a configuration
+        - `flags` (string): flags to be use when building PyNE
+
+
+`run_test`: pull a previous using `pull_container` command, then run the PyNE
+nosetests using the provided `flag`.
+    
+    - `arguments`:
+        - `build` (string): used to identify a configuration
+        - `flags` (string): flags to be use when running the PyNE nosetests
+
+
+`website_build_push`: pull the Python 2.7 build with all the PyNE optionnal
+depedencies `python2_dagmc_pymaob` saved image (using `pull_container` command)
+build the website and push it to the `pyne.github.com` repo.
+    
+    - `arguments`:
+        - `push_option` (string: `test` or `root`): 
```

Comment:
I am very confused based on this specific part of the documentation what the options are? Are they `test`, `root`, and `master`??

Reply:
this par is messed up...